📖 Add GRPOTrainer to README.md (#2713)
huggingface/trl: README.md

trainer.train()
```

### `GRPOTrainer`

`GRPOTrainer` implements a [DeepSeekMath: Pushing the Limits of Mathematical Reasoning in Open Language Models]([https://huggingface.co/papers/2402.14740](https://huggingface.co/papers/2402.03300)) for reinforcement learning. Group Relative Policy Optimization (GRPO) is more performant than PPO and was used to train [Deepseek AI's R1](https://huggingface.co/deepseek-ai/DeepSeek-R1).

```python
from datasets import load_dataset
from trl import GRPOConfig, GRPOTrainer

dataset = load_dataset("trl-lib/tldr", split="train")

> # Define the reward function, which rewards completions that are close to 20 characters

Reviewer: ```suggestion
# Dummy reward function: rewards completions that are close to 20 characters
```
